packages/graphback-cli/src/templates/configTemplates.ts

```diff
@@ -70,21 +62,60 @@ export const askForClient = async (): Promise<boolean> => {
  * Create config file with db info
  */
 export const createConfig = async (database: string, client: boolean) => {
-  const configPath = `${process.cwd()}/graphback.json`
+  const configPath = `${process.cwd()}/.graphqlrc.yml`
 
   const dockerComposePath = `${process.cwd()}/docker-compose.yml`
-  const config = {}
   const [dbConfig, dockerCompose] = getConfig(database)
-  config['db'] = {
-    dbConfig: JSON.parse(dbConfig),
-    database: database
+  const graphqlConfig: IGraphQLConfig = {
+    schema: './src/schema/*.gql',
+    documents: './client/src/graphql/*.gql',
+    extensions: {
+      graphback: {
+        "model": "./model",
+        "crud": {
+          "create": true,
+          "update": true,
+          "findAll": true,
+          "find": true,
+          "delete": false,
+          "subCreate": false,
+          "subUpdate": false,
+          "subDelete": false
+        },
+        "plugins": {
+          "graphback-schema": {
+            "format": "gql",
+            "outputPath": "./src/schema"
+          },
+          "graphback-resolvers": {
+            "format": "ts",
+            "outputPath": "./src/resolvers"
+          }
+        },
+        "dbmigrations": {
```

Comment:
Mixed use of `dbmigrations` and `dbmigrate`

Reply:
Yes. Some of the changes were reverted by mistake. But I guess we need to rename this field. Can you tell me your prefference?

Reply:
Of the two, I prefer `dbmigrations` (perhaps it should be `dbMigrations`)?

`dbmigrate` reads more like an action, not a config.